articles/quickstart/native/flutter/interactive.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Add login to your Flutter app
+default: true
+description: This tutorial demonstrates how to add user login with Auth0 to a Flutter application using the Auth0 Flutter SDK
```

Comment:
I'd suggest to include 'iOS / Android', or 'Android or iOS', etc otherwise the developer could expect it to work in other platforms we currently don't support, like Flutter web.

Reply:
If this description won't show up in the interactive QS, then we should add this bit somewhere else as well.

Reply:
I've added it into the description as well as a note at the top of the article.